src/core.cpp

```diff
@@ -543,28 +543,47 @@ IModelComponent* Core::getComponentByCapability( const string& capabilityName )
 /*! \brief Register a capability as associated with a component.
  *  \param capabilityName The capability of the component to register.
  *  \param componentName The name of the associated component.
+ *  \param warndupe If true (the default) warn when a duplicate capability
+ *                  is declared.  This should only be set to false when being
+ *                  called from registerInput, which sometimes creates duplicates
+ *                  for benign reasons.
  *  \exception h_exception If the componentName was not recognized.
```

Comment:
```suggestion
 *  \exception h_exception If the componentName was not recognized.
 * \note By "capability" we mean any piece of data that the component wants to expose to the model core or other components; this can be an input, an output, or an internal variable.
```